keyboards/xd75/keymaps/skewwhiffy/keymap.c

```diff
@@ -15,36 +15,44 @@
  */
 #include "xd75.h"
 #include "keymap_uk.h"
-// Fillers to make layering more clear
+
 #define _______ KC_TRNS
 
 // Layer shorthand
 #define _CM 0 // Colemak
 #define _DV 1 // Dvorak
 #define _QW 2 // QWERTY
-#define _NB 3 // Numbers
-#define _SY 4 // Symbols
-#define _NAV 5 // Navigation
+#define _NBL 3 // Numbers left
```

Comment:
You may want to consider using an enum here, rather than a bunch of defines.  I tay be simpler in the long run.

Reply:
Done.